modules/oe_paragraphs_media_field_storage/oe_paragraphs_media_field_storage.install

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * OE Paragraphs Media Field Storage install.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Implements hook_install().
+ */
+function oe_paragraphs_media_field_storage_install() {
+  $field_storage = Drupal::entityTypeManager()->getStorage('field_storage_config')->load('paragraph.field_oe_media');
```

Comment:
We need to add here a big fat comment explaining why we are doing this here as opposed to just relying on `config/install`. Let's give a bit of history too, so people can understand well why we did this.